src/components/Home/home.js

```diff
@@ -4,6 +4,7 @@ import './home.scss';
 import { Button , Container, Row, Col } from 'reactstrap';
```

Comment:
Add few comments before writing any code, so that other developer get good understanding of ur code.

Reply:
Fixed